client/components/domains/reskin-side-explainer/index.jsx

```diff
@@ -9,37 +8,44 @@ import './style.scss';
 
 class ReskinSideExplainer extends Component {
 	getStrings() {
-		const { type, eligibleForProPlan, selectedSiteId, translate } = this.props;
+		const { type, translate } = this.props;
 
 		let title;
 		let subtitle;
 		let ctaText;
 
-		// The special case latter is for handling the case in the sign-up flow.
-		// By that time, a site is not available yet, so we can only check the feature flag for now
-		const shouldShowProPlanTitle =
-			eligibleForProPlan || ( selectedSiteId == null && isEnabled( 'plans/pro-plan' ) );
+		const showNewTitle =
+			i18n.hasTranslation( 'Your domain is {{b}}free{{/b}} with WordPress Pro' ) ||
+			'en' === getLocaleSlug();
+
+		const showNewSubtitle =
+			i18n.hasTranslation(
+				'Use the search tool on this page to find a domain you love, then select the {{b}}WordPress Pro{{/b}} plan.{{br}}{{br}}We’ll pay the first year’s domain registration fees for you, simple as that!'
+			) || 'en' === getLocaleSlug();
 
 		switch ( type ) {
 			case 'free-domain-explainer':
-				title = shouldShowProPlanTitle
-					? translate(
+				showNewTitle
+					? ( title = translate( 'Get your domain {{b}}free{{/b}} with WordPress Pro', {
```

Comment:
Should this string be the same as the one [here](https://github.com/Automattic/wp-calypso/pull/62628/files#diff-bd86b71397102cc0a53f776bdd58f259d260237654f752f3ad3eabee38f8160dR18) - `Your domain is {{b}}free{{/b}} with WordPress Pro`?

Reply:
Nice catch, I tweaked the copy after I started working on it. Fix coming.